haproxy_discovery.sh

```diff
@@ -14,6 +14,10 @@
 HAPROXY_SOCK="/var/run/haproxy/info.sock"
 [ -n "$1" ] && echo $1 | grep -q ^/ && HAPROXY_SOCK="$(echo $1 | tr -d '\040\011\012\015')"
 
+# Define variables and perform checks
+CONFIG_FILE="$(dirname $(readlink -f "$0"))/haproxy_config"
+. $CONFIG_FILE
+
```

Comment:
Btw yes. I had idea to get haproxy socket from haproxy config (default one), but services like SELinux can make a huge pain in the a^Whead =]